app/tests/server/test_server.py

```diff
@@ -586,27 +625,27 @@ async def test_user_rate_limiting():
         assert response.json() == {"message": "Rate limit exceeded"}
 
 
-@pytest.mark.asyncio
-async def test_webhooks_rate_limiting():
-    async with AsyncClient(app=app, base_url="http://test") as client:
-        # Mock the webhooks.get_webhook function
-        with patch(
-            "server.server.webhooks.get_webhook",
-            return_value={"channel": {"S": "test-channel"}},
-        ):
-            with patch("server.server.webhooks.is_active", return_value=True):
-                with patch("server.server.webhooks.increment_invocation_count"):
-                    with patch("server.server.sns_message_validator.validate_message"):
-                        # Make 30 requests to the handle_webhook endpoint
-                        payload = '{"Type": "Notification"}'
-                        for _ in range(30):
-                            response = await client.post("/hook/test-id", json=payload)
-                            assert response.status_code == 200
-
-                        # The 31st request should be rate limited
-                        response = await client.post("/hook/test-id", json=payload)
-                        assert response.status_code == 429
-                        assert response.json() == {"message": "Rate limit exceeded"}
+# @pytest.mark.asyncio
```

Comment:
Sorry - I know that this is annoying. Just not sure if this should be here.

Reply:
Good find! I had commented out at the beginning of the refactor to better focus on the function itself.

Adjusting for the changes in the handle_webhook method!